app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+def copy_file(command: str) -> None:
+    # Split the command string into its components
+    parts = command.split()
+
+    # Check if the command has exactly 3 parts
+    if len(parts) != 3:
+        print(
+            "Invalid command format. "
+            "Please provide command in the format: cp old_file new_file"
+        )
+        return
+
+    # Extract the filenames from the command
+    _, old_file, new_file = parts
+
+    # Check if the old_file and new_file are the same
+    if old_file == new_file:
+        print(
+            "The source and destination filenames are the same. "
+            "Nothing to do."
+        )
+        return
```

Comment:
it seems to me, that there will be much less code if u will unpack vars in try except block to validate len of command, and then will write a backward condition to write a file
also u should validate a command)